docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`-` rotki decodes transactions and detects balances related to Gearbox GEAR staking.
 * :feature:`7544` Gearbox protocol events and balances are now decoded and detected on all supported evm chains.
```

Comment:
since you already have the gearbox entry talking about `gearbox protocol events` don't add a new one for staking